### PR TITLE
Add Configuration Information to README

### PR DESCRIPTION
**Pull request by codeblooded**
This pull request is in regards to #163.  It provides README documentation for Quick Configuration.  Let me know if you need anything changed, added, or removed.  Glad to contribute :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
 $ rake templates:uninstall
 ```
 
+## Configuring Quick
```
Comment (modocache) [inline README.md]: nitpick: Add a newline between this header and the paragraph below.

File: README.md [hunk 2]
```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
...
+  // set options of the configuration object
+}
+
+QuickConfigurationEnd
+```
+
+Projects may include several configurations. However, Quick does not make any
+guarantee about the order in which they are executed.
+
+### Adding Global Before and After Filters
```
Comment (modocache) [inline README.md]: nitpick: Newline here as well.

File: README.md [hunk 3]
```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
...
+
+class FinConfiguration: QuickConfiguration {
+  override class func configure(configuration: Configuration) {
+    configuration.beforeEach() {
+      fin = Dorsal.sharedFin()
+      fin.height = 0
+    }
+  }
+}
+```
```
Comment (modocache) [inline README.md]: This is really great. Thanks for the awesome code sample!!

File: README.md [hunk 4]
```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
...
+
+```objc
+// Objective-C
+
+#import <Quick/Quick.h>
+#import "Dorsal.h"
+
+QuickConfigurationBegin(FinConfiguration)
+
++ (void) configure:(Configuration *)configuration {
```
Comment (modocache) [inline README.md]: nitpick: Remove space between `(void)` and `configure:`. The other examples have no space, and I prefer that.

File: README.md [hunk 5]
```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
...
+
+```objc
+// Objective-C
+
+#import <Quick/Quick.h>
+#import "Dorsal.h"
+
+QuickConfigurationBegin(Fin2Configuration)
+
++ (void) configure:(Configuration *)configuration {
```
Comment (modocache) [inline README.md]: nitpick: Remove space here as well.

**Comment by modocache** [comment]
This is awesome--thanks @codeblooded! :+1: 

Sorry it took me a while to review, and sorry for all the nitpicks. But this is great documentation! :100:

**Comment by codeblooded** [comment]
Alright :)  I went through it and I think that I got them all.  If not, just let me know!  Glad to contribute!

**Comment by modocache** [comment]
Thanks! :100: :sparkles:

**Pull request merged**